imagebuildah/build.go

```diff
@@ -220,7 +220,11 @@ func BuildDockerfiles(ctx context.Context, store storage.Store, options BuildOpt
 			// context directory to it.
 			dinfo, err := os.Stat(dfile)
 			if os.IsNotExist(err) {
-				dfile = filepath.Join(options.ContextDirectory, dfile)
+				// if "/workDir" == filepath.Dir("/workDir/Dockerfile")
+				// so don't joint it
+				if options.ContextDirectory != filepath.Dir(dfile) {
+					dfile = filepath.Join(options.ContextDirectory, dfile)
+				}
 				dinfo, err = os.Stat(dfile)
 				if err != nil {
 					return "", nil, errors.Wrapf(err, "error reading info about %q", dfile)
```

Comment:
Could you change this to just return err, not wrapping the error.  THis causes dfile to be listed twice.